test/gui/shared/scripts/pageObjects/Activity.py

```diff
@@ -43,7 +43,7 @@ def clickTab(self, tabName):
         if not tabFound:
             raise Exception("Tab not found: " + tabName)
 
-    def checkFileExist(self, filename):
+    def checkConflictFileExist(self, filename):
```

Comment:
why is the change here?

Reply:
this function checks for conflict files exist or not. Ignored/Blacklisted files can't be checked by that function.